nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/EmptyDocumentTest.java

```diff
@@ -165,4 +168,22 @@ public void testOverrideEmptyDocumentListenerValues() throws IOException {
             assertEquals("dummy subject", subjects.get(0).textValue());
         }
     }
+
+    @Test
+    @ConsoleLogLevelThreshold("ERROR")
```

Comment:
Useless I beleive. I know it's on the other tests but not sure why.

Reply:
Yep, i thought so, but in fact I've just tried to run the other tests without the annotation and I didn't see any error in the console...

Reply:
You're right, behavior has changed for exception within [NXP-21776](https://jira.nuxeo.com/browse/NXP-21776), 4xx exception are no longer logged -> we can remove everything tied to log feature in this test class.

@ThierryCasanova Could you do these removes in a separate commit? I'm in favor to put the _NXP-21776_ in the commit message, wdyt @ataillefer & @troger ?

Reply:
I agree to do the cleanup.
For the commit message, I would go for mentioning NXP-21776 but still making it start with "NXP-26903: ..."